### PR TITLE
adding in a tutorial re enhanced permissions

### PR DESCRIPTION
**Pull request by joshuafernandes**
# Pull request checklist

Use the following template to make sure your PR fits the GoQuorum documentation standard.

## Before creating the PR

Make sure that:

- [x] You've read the [contribution guidelines](https://consensys.net/docs/doctools/).
- [x] You've [previewed your changes locally](https://consensys.net/docs/doctools/en/latest/preview/new-system/#preview-locally).

## Describe the change

Added a tutorial re enhanced permissions to the quickstart

## Impacted parts

For content changes:

- [x] Documentation content
- [ ] Documentation page organization

For tool changes:

- [ ] Build and QA tools (e.g., lint)
- [ ] MkDocs templates
- [ ] MkDocs configuration
- [ ] Python dependencies
- [ ] Node dependencies and JavaScript
- [ ] GitHub integration

## After creating your PR and tests have finished

Make sure that:

- [ ] You've [fixed any issues](https://consensys.net/docs/doctools/en/latest/contribute/fix-cicd-errors/) raised by the tests.
- [ ] You've [previewed your changes on GitHub Pages](https://consensys.net/docs/doctools/en/latest/preview/new-system/#preview-on-github-pages)

### CODE REVIEW

**Review by Ezzahhh — APPROVED** [review]
LGTM

File: docs/tutorials/quorum-dev-quickstart/using-the-quickstart.md [hunk 1]
```diff
@@ -414,6 +415,123 @@ All nodes are validating the same blockchain of transactions, with the private t
...
+
+```bash
+node ./scripts/deploy.js
+```
+
+When this is completed, an output `permission-config.json` is created which contains the addresses of the
+contract and any account addresses that will serve as admins. In this example we've used all accounts, if you
+deploy these contracts in a production network, please select only those which need to be admins.
+
+The file looks like this (but will have different addresses)
```
Comment (antonydenyer) [inline docs/tutorials/quorum-dev-quickstart/using-the-quickstart.md:471]: If you've run the quick start from scratch the address's will be the same!

File: docs/tutorials/quorum-dev-quickstart/using-the-quickstart.md [hunk 2]
```diff
@@ -414,6 +415,123 @@ All nodes are validating the same blockchain of transactions, with the private t
...
+and approve new organizations into the network.
+* subOrgBreadth - Number of sub-organizations that any organization can have.
+* subOrgDepth - Maximum depth of sub-organization hierarchy allowed in the network.
+
+The last step is to copy the above `permission-config.json` into `data` directory of each GoQuorum node and
+restart the nodes. This can be done by running the script
+
+```bash
+./scripts/restartNetwork.sh
+```
```
Comment (antonydenyer) [inline docs/tutorials/quorum-dev-quickstart/using-the-quickstart.md:520]: Can this be put in the root like `./run.sh` `./resume.sh` etc

**Review by antonydenyer — APPROVED** [review]

File: docs/tutorials/quorum-dev-quickstart/using-the-quickstart.md (hunk 1 above)
Comment (joshuafernandes) [reply docs/tutorials/quorum-dev-quickstart/using-the-quickstart.md]: :+1: Fixed that up

File: docs/tutorials/quorum-dev-quickstart/using-the-quickstart.md (hunk 2 above)
Comment (joshuafernandes) [reply docs/tutorials/quorum-dev-quickstart/using-the-quickstart.md:520]: Nice, this is a really good idea. Have updated the quickstart to have this now

**Comment by bgravenorst** [comment]
Closing because this is superseded by https://github.com/ConsenSys/doc.goquorum/pull/223.

**Pull request closed without merging**